### DISCUSSION TITLE
Follow installation process

### DISCUSSION THREAD

**Discussion by avsaponchik** [Klocman/Bulk-Crap-Uninstaller#419] [Ideas]
Kinda curious that there is no kind of discussion about this feature...
Uninstall Tool offers 'follow' process which (no way) follows the installation process to safely roll back when you later need it to.

**Comment by Klocman** [comment]
So it's like an undo for the uninstallation? It might be interesting but it would require that only one application is uninstalled at one time, and there are probably other gotchas to deal with.

**Comment by Klocman** [comment]
After some deliberation I'm not going to pursue this idea.
